portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -1990,6 +1990,30 @@ definition {
 			searchTerm = "Blogs");
 	}
 
+	@description = "This is a use case for LPS-186456 - SearchBar Instance Setting is not updating."
+	@priority = 3
+	test SearchInstanceSettingsAreUpdatedWhenSaved {
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "Instance Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "Search",
+			configurationName = "Search Bar",
+			configurationScope = "Widget Scope");
+
+		Type(
+			locator1 = "SearchBarSuggestions#CHARACTER_THRESHOLD_FOR_DISPLAYING_SUGGESTIONS",
```

Comment:
Avoid using locators outside of their intended place. This adds risk for path updates in the future.

In this case, I would use a `FormField` macro since System and Instance settings are all forms.

Reply:
Also I updated the last assertion with the same path used in `FormFields.editText` since we don't want to use the 
`SearchBarSuggestions#CHARACTER_THRESHOLD_FOR_DISPLAYING_SUGGESTIONS` path